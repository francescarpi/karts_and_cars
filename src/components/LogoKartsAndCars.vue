<template>
  <h1 class="text-white text-5xl">
    <span class="text-red-800 font-faster">Karts</span>
    <span class="ml-2 mr-4 text-xl">&</span>
    <span class="text-yellow-200 font-faster">Cars</span>
  </h1>
</template>
