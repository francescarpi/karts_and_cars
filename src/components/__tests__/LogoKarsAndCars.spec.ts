import LogoKartsAndCars from "../LogoKartsAndCars.vue";
import { mount } from "@vue/test-utils";
import { describe, it, expect } from "vitest";

describe("LogoKartsAndCars", () => {
  it("renders properly", () => {
    const wrapper = mount(LogoKartsAndCars);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
