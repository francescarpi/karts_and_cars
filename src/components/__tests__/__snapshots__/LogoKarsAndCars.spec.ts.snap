// Vitest Snapshot v1

exports[`LogoKartsAndCars > renders properly 1`] = `"<h1 class=\\"text-5xl\\"><span class=\\"text-primary font-faster\\">Karts</span><span class=\\"ml-2 mr-4 text-xl text-black\\">&amp;</span><span class=\\"text-secondary font-faster\\">Cars</span></h1>"`;
